src/modules/advanced-custom-attributes/index.js

```diff
@@ -0,0 +1,191 @@
+/**
+ * External dependencies
+ */
+import { PanelAdvancedSettings } from '~stackable/components'
+import { i18n } from 'stackable'
+import { isUndefined, omit } from 'lodash'
+import striptags from 'striptags'
+
+/**
+ * WordPress dependencies
+ */
+import { addFilter } from '@wordpress/hooks'
+import {
+	Fragment, useState, useEffect, render, unmountComponentAtNode,
+} from '@wordpress/element'
+import { __, sprintf } from '@wordpress/i18n'
+import { TextControl } from '@wordpress/components'
+
+const INVALID_HTML_ATTRIBUTES = [
+	'class',
+	'className',
+	'id',
+]
+
+const INVALID_BLOCK_ATTRIBUTES = [
+	'customAttributes',
+]
+
+const createAddSaveProps = ( extraProps, blockProps ) => {
+	if ( ! Array.isArray( blockProps.attributes.customAttributes ) || blockProps.attributes.customAttributes.length === 0 ) {
+		return extraProps
+	}
+
+	const customAttributes = {}
+	try {
```

Comment:
Use the `try` statement inside the `forEach` so that if it encounters an error, it can still proceed.